src/cljs/athens/listeners.cljs

```diff
@@ -220,17 +220,19 @@
     EventType.BEFOREUNLOAD
     (fn [e]
       (let [synced? @(subscribe [:db/synced])
+            isEditing? @(subscribe [:editing/uid])
```

Comment:
`:editing/uid` signifies only that user has entered editing state, it doesn't mean there are unsaved changes yet.

Reply:
This approach makes sense to me. If i'm in a text area, i'm likely in it with the intention to add data to it. I definitely wouldn't want to be kicked out by accident and would much prefer a popup while i'm writing. 

Would an alternate approach to this issue involve saving at intervals while typing in the block without necessarily having left it? 

I'm not quite sure how i would go about identifying if there are unsaved changes which is why this approach made sense to me. 

This is similar to how logseq does it with command r with a popup dialog confirming with the user, no matter if they have unsaved changes or not.

Reply:
This is a good MVP approach.
We'll need to expose editing dirty state, for next iteration of it.

Thank you for this contribution